.github/workflows/ci.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           mvn -B -q install --file pom.xml -Drun311tests=${{ matrix.run311tests }} -Drun40tests=${{ matrix.run40tests }}
   build-dse:
-    if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
+#    if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
```

Comment:
Yes, that was commented out to get the DSE tests to run. They normally don't run until things are merged to master as this was the best I could do to prevent DSE tests from running on PRs from forks.